assets/src/edit-story/components/panels/test/layerStyle.js

```diff
@@ -31,36 +31,32 @@ describe('Panels/LayerStyle', () => {
   }
 
   it('should render <LayerStyle /> panel', () => {
-    const { getByText } = renderLayerStyle([{ opacity: 100 }]);
-    const element = getByText('Layer');
+    const { getByRole } = renderLayerStyle([{ opacity: 100 }]);
+    const element = getByRole('button', { name: 'Layer' });
     expect(element).toBeDefined();
   });
 
   it('should set opacity to 100 if not set', () => {
-    const { getByText } = renderLayerStyle([{}]);
-    const element = getByText('Opacity');
-    const input = element.getElementsByTagName('input')[0];
+    const { getByRole } = renderLayerStyle([{}]);
+    const input = getByRole('textbox', { name: 'Opacity in percentage' });
     expect(input.value).toStrictEqual('100%');
   });
 
   it('should set opacity to 100 if set to 0', () => {
-    const { getByText } = renderLayerStyle([{ opacity: 0 }]);
-    const element = getByText('Opacity');
-    const input = element.getElementsByTagName('input')[0];
+    const { getByRole } = renderLayerStyle([{ opacity: 0 }]);
+    const input = getByRole('textbox', { name: 'Opacity in percentage' });
     expect(input.value).toStrictEqual('100%');
   });
 
   it('should set opacity to 49 if set to 49', () => {
-    const { getByText } = renderLayerStyle([{ opacity: 49 }]);
-    const element = getByText('Opacity');
-    const input = element.getElementsByTagName('input')[0];
+    const { getByRole } = renderLayerStyle([{ opacity: 49 }]);
+    const input = getByRole('textbox', { name: 'Opacity in percentage' });
     expect(input.value).toStrictEqual('49%');
```

Comment:
Looking at this one, it really should be done with [`toHaveDisplayValue`](https://github.com/testing-library/jest-dom#tohavedisplayvalue). But that's definitely outside the scope of this PR!